.eslintrc.js

```diff
@@ -56,6 +56,9 @@ module.exports = {
         "@typescript-eslint/triple-slash-reference": "error",
         "@typescript-eslint/type-annotation-spacing": "error",
         //"@typescript-eslint/unified-signatures": "error",
+        "@typescript-eslint/no-floating-promises": "error",
+        "@typescript-eslint/await-thenable": "error",
```

Comment:
`"@typescript-eslint/await-thenable": "error"` this rule didn't change anything in the code. Was there a specific reason to add it?

Reply:
It helps cleaning up the code when doing refactoring: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/await-thenable.md

Reply:
For future?

Reply:
Right

Reply:
we can keep it, I was wondering if it was fixing anything for tests right now.

Reply:
This one doesn't really affect execution. Sorry for the confusion.